plugin/db/util/driverutil.go

```diff
@@ -306,6 +306,9 @@ func EndMigration(ctx context.Context, executor MigrationExecutor, startedNs int
 
 // Query will execute a readonly / SELECT query.
 func Query(ctx context.Context, dbType db.Type, sqldb *sql.DB, statement string, limit int) ([]interface{}, error) {
+	// Limit SQL query result size.
+	statement = getStatementWithResultLimit(statement, limit)
+
```

Comment:
With this, we no longer need break on l402?

Reply:
Done.